### ISSUE TITLE
Server-side execution on my on server?

### ISSUE THREAD

**Issue by carstenf** [thebaselab/codeapp#913]
I would like to use an external server (AWS EC2, Azure..) to install Python and run the code on that instance.
As you are mention, you use Server-side execution.
Theoretically this should be already available.

**Comment by bummoblizard** [comment]
You can do this by connecting to SSH instances you have. To learn more about this: https://code.thebaselab.com/guides/connecting-to-a-remote-server-ssh-ftp

**Comment by carstenf** [comment]
Yes, that I did, but if I run a python code stored in my document folder, it picks up the pre installed python Version from your app. 
How I can change the settings, that the python version installed on my AWS EC2 is used?

**Comment by bummoblizard** [comment]
Code App uses an open-source code-execution server called Judge0. Although currently there isn't a way to configure the server endpoint in app, you could change them in this file and build the app yourself. https://github.com/thebaselab/codeapp/blob/25c470c8a33f892a4a013c01acb545437facf3ef/CodeApp/Constants/Secrets.swift#L1

**Comment by bummoblizard** [comment]
> Thanks
> 
> I looked into it, but I believe I’m not able to get that running.
> 
> Is there an option that this could be included in the app?
> 
> -> run code in the server which is actually connected.

Here the steps you need to do:
- Run Judge0 on a server that is accessible from your phone
- Clone this repository and change the file I mentioned and point it to your own server 
- Download Xcode and build the app on your phone

Having the server configurable in app is possible too but is not currently available. If this is too much hassle, I'd suggest you to use the SSH feature instead.
